### ISSUE TITLE
resnet_v2_101 does not woking well as expected

### ISSUE THREAD

**Issue by wjb123** [tensorflow/models#2008]
When I use the pretrained the model resnet_v2_101 in the slim of tensorflow, the classification results of the input image are not expected . the code is as follow:

```
    with tf.Graph().as_default():
        test_imag = '../../cat.jpg'
        image = tf.image.decode_jpeg(tf.read_file(test_imag), channels=3)
        processed_image = inception_preprocessing.preprocess_image(image, 299, 299, is_training=False)
        processed_images  = tf.expand_dims(processed_image, 0)
        with slim.arg_scope(resnet_v2.resnet_arg_scope()):
            logits, _ = resnet_v2.resnet_v2_101(processed_images, num_classes=1001, is_training=False)
        probabilities = tf.nn.softmax(logits)
        init_fn = slim.assign_from_checkpoint_fn('../../tf_models/resnet_v2_101/resnet_v2_101.ckpt', slim.get_model_variables('resnet_v2_101'))
        with tf.Session() as sess:
            init_fn(sess)
            org, pro, probabilities = sess.run([image, processed_images, probabilities])
            probabilities = probabilities[0, 0:]
            sorted_inds = [i[0] for i in sorted(enumerate(-probabilities), key=lambda x:x[1])]
        names = imagenet.create_readable_names_for_imagenet_labels()
        for i in range(5):
            index = sorted_inds[i]
            print('Probability %0.2f%% => [%s]' % (probabilities[index] * 100, names[index]))
```

the results are as follows:
```
Probability 12.36% => [Egyptian cat]
Probability 11.90% => [tiger cat]
Probability 4.80% => [kit fox, Vulpes macrotis]
Probability 4.65% => [tabby, tabby cat]
Probability 2.57% => [red fox, Vulpes vulpes]
```
Can you give some advises ?

**Comment by asimshankar** [comment]
If I understand your question correctly, you're asking why the classification for a given image is not what you expect it to be? If so, that is something that is probably out of scope as far as being a bug report or feature request is concerned :).  In general, explaining why neural networks make particular decisions is an active area of research and even the best image classification networks have less than perfect accuracy. See https://research.googleblog.com/2016/08/improving-inception-and-image.html for example.
